### ISSUE TITLE
Open Script By Type

### ISSUE THREAD

**Issue by Mefodei** [alelievr/NodeGraphProcessor#48]
I'm notice the following source code

```csharp
public void OpenNodeScript()
{
	var scriptPath = NodeProvider.GetNodeScript(nodeTarget.GetType());

#pragma warning disable CS0618 // Deprecated function but no alternative :(
	if (scriptPath != null)
		InternalEditorUtility.OpenFileAtLineExternal(scriptPath, 0);
#pragma warning restore CS0618
}
```
But you can do it by this way

```csharp
 public static bool OpenScript(Type type,params string[] folders)
        {
            var typeName  = type.Name;
            var filter    = $"t:script {typeName}";
            
            var assets = AssetDatabase.FindAssets(filter, folders);
            var assetGuid = assets.FirstOrDefault(
                    x => string.Equals(typeName,
                        Path.GetFileNameWithoutExtension(x.AssetGuidToPath()),
                        StringComparison.OrdinalIgnoreCase));
            
            if (string.IsNullOrEmpty(assetGuid))
                return false;
            
            var asset = AssetDatabase.LoadAssetAtPath<Object>(assetGuid.AssetGuidToPath());
            if (asset == null)
                return false;
            return AssetDatabase.OpenAsset(asset.GetInstanceID(), 0, 0);
        }
```

**Comment by alelievr** [comment]
Ah yes, that true. I forgot this function.

I'll replace the deprecated one by the OpenAsset, thanks :)

**Comment by alelievr** [comment]
Fixed in c1e852c338342e585e7d16b7c2f499a4fa20b3b4 and published in package 0.6.0 :)
